dapp/sentry.edge.config.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+// This file configures the initialization of Sentry for edge features (middleware, edge routes, and so on).
+// The config you add here will be used whenever one of the edge features is loaded.
+// Note that this config is unrelated to the Vercel Edge Runtime and is also required when running locally.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs';
+
+import { IS_PROD, SENTRY_DSN } from './sentry.common.config.mjs';
+
+Sentry.init({
+    enabled: IS_PROD && Boolean(SENTRY_DSN),
+    dsn: SENTRY_DSN,
+
+    // Define how likely traces are sampled. Adjust this value in production, or use tracesSampler for greater control.
+    tracesSampleRate: 0, // Edge is not traced
+
+    // Setting this option to true will print useful information to the console while you're setting up Sentry.
+    debug: false,
```

Comment:
```suggestion
    debug: false,
    
    ignoreErrors: ['AbortError', 'Network request failed', 'NetworkError'],
```

We could ignore some error that is not critical